diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/bitstream-tests.py pulp_rpm/test/unit/rhsm_tests/bitstream-tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/bitstream-tests.py	2012-10-30 11:13:35.055699704 -0400
+++ pulp_rpm/test/unit/rhsm_tests/bitstream-tests.py	2012-10-30 11:50:45.218440354 -0400
@@ -15,10 +15,10 @@ import os
 import unittest
 import zlib
 
-from rhsm.bitstream import GhettoBitStream
+from pulp_rpm.repo_auth.rhsm.bitstream import GhettoBitStream
 
 DATA = os.path.join(os.path.dirname(os.path.abspath(__file__)),
-                    'entitlement_data.bin')
+    'entitlement_data.bin')
 entitlement_data = open(DATA).read()
 decompresser = zlib.decompressobj()
 decompresser.decompress(entitlement_data)
diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/certificate2-tests.py pulp_rpm/test/unit/rhsm_tests/certificate2-tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/certificate2-tests.py	2012-10-30 11:13:35.004699985 -0400
+++ pulp_rpm/test/unit/rhsm_tests/certificate2-tests.py	2012-10-30 11:50:45.218440354 -0400
@@ -14,12 +14,11 @@
 #
 
 import unittest
-import os
 from datetime import datetime
 
 import certdata
-from rhsm.certificate import create_from_pem, CertificateException
-from rhsm.certificate2 import *

+from pulp_rpm.repo_auth.rhsm.certificate2 import *
 
 from mock import patch
 
@@ -99,7 +98,7 @@ class V1CertTests(unittest.TestCase):
         self.assertFalse(self.ent_cert.check_path('/foo/'))
         self.assertFalse(self.ent_cert.check_path('/foo/path/'))
 
-    @patch('rhsm.certificate2.EntitlementCertificate._validate_v1_url')
+    @patch('pulp_rpm.repo_auth.rhsm.certificate2.EntitlementCertificate._validate_v1_url')
     def test_download_url_identification(self, mock_validate):
         # there are 4 OIDs in the testing cert that should be checked, and
         # many others that should not. This verifies that exactly 4 OIDs get
Binary files /home/mhrivnak/git/python-rhsm/test/unit/.certificate2-tests.py.swp and pulp_rpm/test/unit/rhsm_tests/.certificate2-tests.py.swp differ
diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/certificate-tests.py pulp_rpm/test/unit/rhsm_tests/certificate-tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/certificate-tests.py	2012-10-22 17:28:55.418626319 -0400
+++ pulp_rpm/test/unit/rhsm_tests/certificate-tests.py	2012-10-30 11:50:45.218440354 -0400
@@ -15,7 +15,7 @@
 
 import unittest
 
-from rhsm.certificate import Key
+from pulp_rpm.repo_auth.rhsm.certificate import Key
 
 
 class KeyTests(unittest.TestCase):
diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/config-tests.py pulp_rpm/test/unit/rhsm_tests/config-tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/config-tests.py	2012-10-22 17:28:55.419626313 -0400
+++ pulp_rpm/test/unit/rhsm_tests/config-tests.py	2012-10-30 11:50:45.219440349 -0400
@@ -16,7 +16,7 @@
 import unittest
 from tempfile import NamedTemporaryFile
 
-from rhsm.config import RhsmConfigParser
+from pulp_rpm.repo_auth.rhsm.config import RhsmConfigParser
 
 TEST_CONFIG = """
 [foo]
diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/connection-tests.py pulp_rpm/test/unit/rhsm_tests/connection-tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/connection-tests.py	2012-10-22 17:28:55.419626313 -0400
+++ pulp_rpm/test/unit/rhsm_tests/connection-tests.py	2012-10-30 11:50:45.219440349 -0400
@@ -13,14 +13,13 @@
 # in this software or its documentation.
 #
 
+from datetime import date
 import unittest
 
-from rhsm.connection import UEPConnection, ConnectionException, ConnectionSetupException, \
-        BadCertificateException, RestlibException, GoneException, NetworkException, \
-        RemoteServerException
-
 from mock import Mock
-from datetime import date
+
+from pulp_rpm.repo_auth.rhsm.connection import UEPConnection, ConnectionException, ConnectionSetupException, \
+        BadCertificateException, RestlibException, GoneException
 
 
 class ConnectionTests(unittest.TestCase):
diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/huffman-tests.py pulp_rpm/test/unit/rhsm_tests/huffman-tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/huffman-tests.py	2012-10-30 11:13:34.828700952 -0400
+++ pulp_rpm/test/unit/rhsm_tests/huffman-tests.py	2012-10-30 11:50:45.219440349 -0400
@@ -13,7 +13,7 @@
 
 import unittest
 
-from rhsm.huffman import HuffmanNode
+from pulp_rpm.repo_auth.rhsm.huffman import HuffmanNode
 
 
 class TestHuffmanNode(unittest.TestCase):
diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/pathtree-tests.py pulp_rpm/test/unit/rhsm_tests/pathtree-tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/pathtree-tests.py	2012-10-30 11:13:35.055699704 -0400
+++ pulp_rpm/test/unit/rhsm_tests/pathtree-tests.py	2012-10-30 11:50:45.219440349 -0400
@@ -15,13 +15,14 @@ from collections import deque
 import os
 import unittest
 
-from rhsm.bitstream import GhettoBitStream
-from rhsm.huffman import HuffmanNode
-from rhsm.pathtree import PathTree, PATH_END
+from pulp_rpm.repo_auth.rhsm.bitstream import GhettoBitStream
+from pulp_rpm.repo_auth.rhsm.huffman import HuffmanNode
+from pulp_rpm.repo_auth.rhsm.pathtree import PathTree, PATH_END
 
 DATA = os.path.join(os.path.dirname(os.path.abspath(__file__)),
                     'entitlement_data.bin')
 
+
 class TestPathTree(unittest.TestCase):
     def test_get_leaf_from_dict(self):
         codes = {'1010' : 'abc'}
diff -rupN /home/mhrivnak/git/python-rhsm/test/unit/version_tests.py pulp_rpm/test/unit/rhsm_tests/version_tests.py
--- /home/mhrivnak/git/python-rhsm/test/unit/version_tests.py	2012-10-22 17:28:55.419626313 -0400
+++ pulp_rpm/test/unit/rhsm_tests/version_tests.py	2012-10-30 11:50:45.219440349 -0400
@@ -13,9 +13,10 @@
 # in this software or its documentation.
 #
 
-from rhsm.version import Versions
 import unittest
 
+from pulp_rpm.repo_auth.rhsm.version import Versions
+
 NOT_COLLECTED = "non-collected-package"
 
 # NOTE: Because the super class will only initialize version data
